libs/ymir-core/src/ymir/util/date_time.cpp

```diff
@@ -17,15 +19,21 @@ sint64 delta_to_host(const DateTime &dateTime) {
     tm.tm_hour = dateTime.hour;
     tm.tm_min = dateTime.minute;
     tm.tm_sec = dateTime.second;
-    return mktime(&tm) - time(nullptr);
+    return std::mktime(&tm) - std::time(nullptr);
 }
 
 DateTime from_timestamp(sint64 secondsSinceEpoch) {
-    tm tm{};
+    const std::chrono::seconds epochSeconds(secondsSinceEpoch);
+    // std::chrono::system_clock is unix-time as of C++20
+    const std::chrono::sys_seconds unixSeconds(epochSeconds);
+
+    const std::time_t timeValue = std::chrono::system_clock::to_time_t(unixSeconds);
```

Comment:
These variables are only used in the `#else` branch below. Move them to avoid compiler warnings about unused variables on Windows.

Reply:
Looks like technically windows should be using the `time_t` variable here too for [localtime_s](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/localtime-s-localtime32-s-localtime64-s?view=msvc-170). Will fix!

Reply:
LGTM now. Thanks again!